pkg/apis/monitoring/v1/types.go

```diff
@@ -118,6 +118,8 @@ type PrometheusSpec struct {
 	ScrapeInterval string `json:"scrapeInterval,omitempty"`
 	// Interval between consecutive evaluations.
 	EvaluationInterval string `json:"evaluationInterval,omitempty"`
+	// Minimum amount of time to wait before resending an alert to Alertmanager.
+	AlertResendDelay string `json:"alertResendDelay,omitempty"`
```

Comment:
as there are multiple `-rules.alert` flags, could you add two config structs for this?

basically so we end up with

```
rules:
  alert:
    resendDelay: ...
```